srcpkgs/ydotool/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'ydotool'
+pkgname=ydotool
+version=0.2.0
+revision=1
+build_style=cmake
+hostmakedepends="git scdoc"
+short_desc="Generic command-line automation tool, works on Wayland and X11"
+maintainer="noarchwastaken <noarch@n0ar.ch>"
+license="AGPL-3.0-or-later"
+homepage="https://github.com/ReimuNotMoe/ydotool"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=2311b003d2ff383f3348f17101f0df74f56616d530d66d0a014a52ba85a5dcf1
+
+system_accounts="_ydotoold"
+_ydotoold_groups="input"
+
+# patches a dependency pulled by CPM, so it must be done after configure
+post_configure() {
+	patch -sl -Np0 -i ${FILESDIR}/iodash-musl.patch 2>/dev/null
+}
```

Comment:
CPM shouldn't be allowed to pull in dependencies. You need to manually download the tarball or package the dependency.